Makefile

```diff
@@ -87,4 +96,4 @@ coveralls:
 
 circle-ci:
 	@${cidocker} make install-deps build-local check-local coveralls
-	make integration-tests
+	make integration-tests clean
```

Comment:
I don't think we need to necessarily remove for `circle-ci` since all the test stuff in circleci are automatically destroyed